source/gsoc/_index.md

```diff
@@ -136,35 +136,35 @@ date indicated.
 <a name="GSoC-StudentApplicationandSelection"></a>
 ## Student Application and Selection
 
-  - 2023-02-22: Potential GSoC contributors discuss application ideas with mentoring organizations
```

Comment:
the dates are different for each year

plus we are shifting original GSoC dates (making them 1-2 days earlier) to have enough time to ping mentors ....

Reply:
How about remove the old dates and leave a TBD here?

Reply:
well, I would VOTE for having up-to-date dates (to have something for the reference)